ClassPaulita/GuessTheCardGame.cs

```diff
@@ -26,39 +160,60 @@ private bool UserChoseToQuit(char userChoice)
             return userChoice == 'q' || userChoice == 'Q';
         }
 
-        private bool IsUserChoiceValid(char userChoice)
-        {                      
+        private char GetUserGuess(Card previousCard)
+        {
+            Console.Write($"\n\n\tIs the next card greater than {previousCard}(y or n)? ");
+            char userGuess = Convert.ToChar(Console.ReadLine());
 
-            if (UserChoseToPlay(userChoice) || UserchoseToSeeInstructions(userChoice) || UserChosetoSeeBoard(userChoice) || UserChoseToQuit(userChoice))
+            while (!IsUserGuessValid(userGuess))
             {
-                return true;
+                Console.WriteLine("\n\tYour answer is not valid. Please try again!");
+                Console.Write($"\n\n\tIs the next card higher or lower than {previousCard}(y or n)? ");
+                userGuess = Convert.ToChar(Console.ReadLine());
             }
 
-            return false;
+            return userGuess;
         }
 
-        private char GetUserMenuChoice()
+        private void AddUserScoreToBoard(int userScore)
         {
+            Console.WriteLine($"Congratulations! You made it to the Leader Board.\n{_spacing}Please enter your userName (10 characters max):\n\n{_spacing}");
+            string userName = Console.ReadLine();
 
-            string menu = $"{spacing}Press P To Play" +
-                          $"{spacing}Press I For Instructions" +
-                          $"{spacing}Press B To See The High Score Board" +
-                          $"{spacing}Press Q To Quit";
+            _scoreBoard.Add(new KeyValuePair<int, string>(userScore, userName));
 
-            Console.WriteLine(menu);
+            while(!IsUserNameValid(userName))
+            {
+                Console.WriteLine($"\n{_spacing}You entered an invalid name. Please try again.\n{_spacing}Please enter your userName (10 characters max):\n\n{_spacing}");
+            }
 
-            char userChoice = Convert.ToChar(Console.ReadLine());
+            _scoreBoard.Sort((x, y) => y.Key.CompareTo(x.Key));
 
-            while (!IsUserChoiceValid(userChoice))
+            if (_scoreBoard.Count > 5)
             {
-                Console.WriteLine("\n\tOoooops! You pressed an invalid option. Please try again.");
+                int linesToRemove = _scoreBoard.Count - 5;
+                int counter = 0;
 
-                Console.WriteLine(menu);
+                while (counter < linesToRemove)
```

Comment:
This would be simplier to just use the count:
```C#
while (_scoreBoard.Count > 5)
{
    // Remove one
}
```

Reply:
Ok. Fixed it

Reply:
I'm not seeing the fix, but you marked this as resolved

Reply:
I see it now. Looks good.